devices.js

```diff
@@ -6409,6 +6409,72 @@ const devices = [
         fromZigbee: [fz.on_off],
         toZigbee: [tz.on_off],
     },
+
+    // Drayton Wiser
+    {
+        zigbeeModel: ['iTRV'],
+        model: 'WV704R0A0902',
+        vendor: 'Dryton Wiser',
```

Comment:
Rename this to 'Dryton/Eberle' ?

Reply:
But what if we have other local brands? Perhaps use Schneider Electric?

Reply:
Schneider Electric is what the valve reports via zigbee too

Reply:
Yes, Schneider Electric might be better! We can update the docs to list the local brand names perhaps?

Reply:
Fixed